engineering_tools/rosbag_matching/README.txt

```diff
@@ -0,0 +1,13 @@
+Rosbag Matcher
+
+Matches a given rosbag to a log directory in a given logfile by examining timestamps in the directories and renaming it to match the bag if the difference in timestamps is a minute or less. 
```

Comment:
This statement is a little confusing, in that it appears to be casually inconsistent between referring to files vs directories.  It is unfortunate that ROS puts all the log dirs into a directory named "logfiles"!  But it is worth explaining that (and using that exact name).  Probably good to state the assumption that the script is run from a directory that contains one or more .bag files and a directory named logfiles, under which are all the log directories (each of which contains a bunch of files from a single run).